### ISSUE TITLE
GPU Transfer

### ISSUE THREAD

**Issue by benedekrozemberczki** [AstraZeneca/chemicalx#65]
- Accelerating training by transfer to GPU.

**Comment by GavEdwards** [comment]
@mughetto @benedekrozemberczki  A simple option for flexible GPU support is to use [Accelerate](https://github.com/huggingface/accelerate).

![image](https://user-images.githubusercontent.com/24434382/152422357-28055a96-f4c9-4183-a574-c1a83be15e5b.png)

I played around with a quick implementation for chemicalx and it's very easy, only changed a few lines of code. I tested on CPU and single GPU and it works well so far. GPU speeds up training and gets high utilisation, e.g the deepddi example is so much faster with GPU enabled 🏎.

If you're interested in this approach I can make an initial PR?
